nestkernel/connection_manager.cpp

```diff
@@ -1225,33 +1225,53 @@ nest::ConnectionManager::get_connections(
 
       std::deque< ConnectionID > conns_in_thread;
 
+      // Split targets into node- and device-vectors.
+      std::vector< index > target_nodes;
```

Comment:
I am not happy with the name `target_nodes`, since devices are also nodes. And the container stores GIDs, not nodes. We do not have an established name for non-device nodes in NEST, unfortunately, but generally they all are neurons.  
```c++
std::vector< index > target_neuron_gids;
std::vector< index > target_device_gids;
```